src/main/clojure/deps/analysis.clj

```diff
@@ -86,9 +96,224 @@
   (->> bundles
        (b/partition-bundles-by-id)
        (map-indexed
-           (fn [idx bundles]
-             (let [cluster (keyword (str "cluster_" idx))]
-               (layer-create-node cluster (str "test_" idx) (map :name bundles)))))))
+         (fn [idx bundles]
+           (let [cluster (keyword (str "cluster_" idx))]
+             (layer-create-node cluster (str "test_" idx) (map :name bundles)))))))
+
+;; ----------------------------------------------------------------------
+;; # HTML Output
+;;
+;; Exporting Loom graphs for an interactive HTML document.
+
+(def ddf-home (System/getProperty "ddf.home"))
+(def viz-file (str ddf-home "/graphs/viz.html"))
```

Comment:
How do these get deployed to DDF?

Reply:
The `viz-file` is just an output directory for where we write the HTML exports that can be opened and manipulated. Similar to how we were using a temp directory for dorothy / graphviz renders. 

No deployment necessary. Executing a function on the REPL will cause the write or overwrite to occur.